addons/resource.language.en_gb/resources/strings.po

```diff
@@ -7766,11 +7775,11 @@ msgid "Already started recording on this channel"
 msgstr ""
 
 msgctxt "#19035"
-msgid "%s could not be played. Check the log for details."
+msgid "%s couldn't be played. Check the log for details."
 msgstr ""
 
 msgctxt "#19036"
-msgid "This recording cannot be played. Check the log for details."
+msgid "Cant't play this recording. Check the log for details."
 msgstr ""
```

Comment:
I also prefer old wording - much nicer to read (It's not about cannot vs can't but the structure of the sentence)